src/JsonSchema/Constraints/CollectionConstraint.php

```diff
@@ -136,7 +137,7 @@ protected function validateItems(&$value, $schema = null, JsonPointer $path = nu
 
             // Treat when we have more schema definitions than values, not for empty arrays
             if (count($value) > 0) {
-                for ($k = count($value); $k < count($schema->items); $k++) {
+                for ($k = count($value); $k < count($schema->items); ++$k) {
```

Comment:
This seems dangerous if the auto-fixer made this change; they aren't syntactically equivalent. It doesn't matter in this particular case, but if the same change is made in a place that relies on the return value of `++`, then it'll alter the logic.

$var++; //increments `$var` and returns the *pre*-increment value
++$var; // increments `$var` and returns the *post*-increment value